reference/course.v1.yaml

```diff
@@ -111,163 +121,118 @@ paths:
-        name: studentId
-        description: Unique student identifier
-        required: true
-      - in: query
-        name: start
-        description: 'Start point for pagination of results, defaults to 0,'
-        schema:
-          type: integer
-          format: int32
-        examples:
-          default:
-            value: 0d
-            summary: The start point for pagination
-      - in: query
-        name: limit
-        description: 'Limit of number of results returned by page, defaults to 20 with max 100.'
-        schema:
-          type: integer
-          maximum: 100
-          format: int32
-        examples:
-          default:
-            value: 20
-            summary: The default value if none is provided
-          max:
-            value: 100
-            summary: The largest recommended page size
-      - in: query
-        name: schoolYear
-        description: School year the results were recorded in
-        schema:
-          type: string
-        examples:
-          default:
-            value: 2020-2021
-            summary: The default value if none is provided
-    get:
-      summary: 'Get XAPI statements related to a specific student, newest statements first'
-      tags: []
-      responses:
-        '200':
-          description: OK
-          content:
-            application/json:
-              schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/Statement'
-      operationId: get-user-statements
-      description: Get statements for a specific user
+                type: object
+                properties: {}
+              examples:
+                example-1:
+                  value: {}
+      operationId: get-course-cmi5
+      description: |-
+        Get the meta data of a specific course by stream ID - 
+        The meta data contains information abou the course, including which chapters, paragraphs and assignable learning units are available, including links to the publishers playing envirionment so the student start that unit.
+        The LMS can use this to personalise the course per student and add or skip assignable units. This structure and information is also the base for exchanging progress.
+        NOTE: the course content itself is not part of the specification, it describes the meta information and how to navigate to the right point in the course, but not the course itself.
+
+        The CMI5 standard also provides a protocol to launch assignable units in a browser and get progress on thse units. This is not part of the SEM project. The format is used to exchange meta information, not to enforce runtime. For the moment it is expected that the url's in the content work without a need to setup sessions etc. each publisher is responsible for creating url's in such a way that single signof is enforced. example: https://toegang.noordhoff.nl/8717927999997?deepLinkId=e19ffbe3-2411-4d1f-8aa6-14fc43c2142a
```

Comment:
Do we need to state that Single Sign On between parties is supported and the mechanism (to my knowledge this is now the standard in the Dutch Secondary Education Eco-system; but might be good to state it explicit)